README.md

```diff
@@ -106,6 +106,8 @@ The options object is optional and has the following keys:
   interface](lib/interfaces/png-processor.js),
 - **rootDirectory** - the path where the screenshots and diffs will be saved;
   defaults to `visual-tests`.
+- **acceptFirstBaseline** - whether the test should pass if the selector is
+  tested for the first time. Defaults to `true`.
```

Comment:
Also add: `The baseline will be saved to the file system regardless of this option`.